### ISSUE TITLE
Collect projects we are major contributors to

### ISSUE THREAD

**Issue by hennevogel** [SUSE/suse.github.io#2]

**Comment by cyphar** [comment]
* [Portus](https://github.com/SUSE/Portus)
* [zypper-docker](https://github.com/SUSE/zypper-docker)

**Comment by ancorgs** [comment]
* [Jangouts](https://github.com/jangouts/jangouts)
* [Machinery](http://machinery-project.org/)

**Comment by ancorgs** [comment]
* [openQA](https://os-autoinst.github.io/openQA/)

**Comment by ancorgs** [comment]
The line between this and #1 is not clear. But since we already had Portus and zypper-docker here, I continued adding stuff only here.

**Comment by cyphar** [comment]
@ancorgs I agree with @cornelius that this line is not very clear. But I believe that anything that is not vaguely a distribution should be placed here (just because we started a project doesn't mean that the project **itself** is a product -- we make products using free software projects).

**Comment by ancorgs** [comment]
I also agree. But I was not talking about this versus #7 (projects vs products) but about this versus #1 (projects we drive vs projects we are major contributors).

**Comment by cornelius** [comment]
It doesn't matter too much, where we collect the project, we can also sort them later. But I think it makes sense to separate projects SUSE has started or SUSE maintains (this would be the projects SUSE drives) and project where we contribute, but where we don't play a dominant role, such as:

* Linux Kernel
* OpenStack
* Ceph
* gcc
* GNOME
* KDE
* LibreOffice
